Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1

```diff
@@ -395,15 +405,22 @@ function Enable-SecureFTPDeploymentForAppServices
             $isSecureFTPEnabledOnAllNonProdSlots = $true;
```

Comment:
Should we rename this attribute also to $AreNonProductionSlotsCompliant instead of $isSecureFTPEnabledOnAllNonProdSlots?

Reply:
done